config/yugabyte/regression_pipelines/orderby_workloads/postgres/ORDG7_orderby_on_hash_partitioned_table.yaml

```diff
@@ -75,44 +80,91 @@ microbenchmark:
               run:
                   - name: yb_ordby_bigint_non_indexed_column
                     weight: 100
                     queries:
                         - query: select col_bigint_id_1 from Indexed1M_1 where col_bigint_id_1>1 order by col_bigint_id_1
             - workload: ORDG7_ordby_bigint_hash_secondary_indexed_column
+              customTags: schematype=regular,partition=hash,skey=hash,cardinality=single,projection=hashskey,queryshape=orderby,orderbyon=hashskey
               run:
                   - name: ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
                         - query: select col_bigint_id_2 from Indexed1M_1 order by col_bigint_id_2
             - workload: ORDG7_ordby_bigint_hash_secondary_indexed_with_filter_column
+              customTags: customer=1,schematype=regular,partition=hash,skey=hash,cardinality=single,projection=hashskey,filtercount=one,filteron=hashskey,filtertype=comparision,queryshape=orderby,orderbyon=hashskey
               run:
                   - name: yb_ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
                         - query: select col_bigint_id_2 from Indexed1M_1 where col_bigint_id_2>1 order by col_bigint_id_2
             - workload: ORDG7_ordby_bigint_range_secondary_indexed_column
+              customTags: schematype=regular,partition=hash,skey=asc,cardinality=single,projection=indexed,queryshape=orderby,orderbyon=rangeskey
               run:
                   - name: ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
-                        - query: select col_bigint_id_4 from Indexed1M_1 order by col_bigint_id_4
+                        - query: select col_bigint_id_3 from Indexed1M_1 order by col_bigint_id_3
             - workload: ORDG7_ordby_bigint_range_secondary_indexed_with_filter_column
+              customTags: customer=1,schematype=regular,partition=hash,skey=asc,cardinality=single,projection=indexed,filtercount=one,filteron=rangeskey,filtertype=comparision,queryshape=orderby,orderbyon=rangeskey
               run:
                   - name: yb_ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
-                        - query: select col_bigint_id_4 from Indexed1M_1 where col_bigint_id_4>1 order by col_bigint_id_4
\ No newline at end of file
+                        - query: select col_bigint_id_3 from Indexed1M_1 where col_bigint_id_3>1 order by col_bigint_id_3
+
+            - workload: ORDG7_ordby_bigint_hash_pkey_with_filter_column_desc
+              customTags: customer=1,schematype=regular,partition=hash,pkey=composite,cardinality=multiple,projection=hashpkey,filtercount=one,filteron=hashpkey,filtertype=comparision,queryshape=orderby,orderbyon=hashpkeydesc
+              run:
+                  - name: yb_ordby_bigint_non_indexed_column
```

Comment:
Query names are same for newly added queries, they should have unique names.

Reply:
Same as above.